tools/server-test/test-runner.ts

```diff
@@ -0,0 +1,96 @@
+const runfiles = require(process.env['BAZEL_NODE_RUNFILES_HELPER']!);
```

Comment:
For a follow-up: can we add a small top-level comment that explains what is done as part of this runner?